dfget/core/core.go

```diff
@@ -176,6 +176,7 @@ func downloadFile(cfg *config.Config, supernodeAPI api.SupernodeAPI,
 	}
 
 	timeout := netutils.CalculateTimeout(cfg.RV.FileLength, cfg.MinRate, config.DefaultMinRate, 10*time.Second)
+	logrus.Error(timeout)
```

Comment:
Delete it or perfect it/

Reply:
This is used for debugging 🤣  I will delete it, thank you.